backend/src/routes/login.route.ts

```diff
@@ -16,25 +22,70 @@ export class LoginRoute {
 
   private initializeRoutes() {
     this.router.post(this.path, async (req, res) => {
-      const requestValidity = this.validateRequest(req.body);
-
-      if (requestValidity.error)
-        return res.status(404).send(requestValidity.error.message);
+      const { error } = this.validateRequest(req.body);
```

Comment:
This doesn't return an error but an object with error attribute as well as the validated message result, e.g. here it also applies lowercase to the returned values achieving case insensitivity with the use of the returned email in the findOne method on line 29. Think my naming could be better here but not sure is there another way to do this?

Reply:
Yeah I think that makes sense, JOI should provide some handy messages for why the validation failed.
It'll return a more complex object with the name of the field, and which validator was tripped up, with a message on how to fix it,
birth_year
   an integer between 1900 and 2013

I think we should keep it, just maybe rename it errorMessage to avoid any confusion?

Reply:
More so, in the case of no validation error, JOI also returns the values with any conversion e.g. conversion of email to lowercase in whatevervariable.value.email - Like it's returning either an error message, or the validated request with a lowercase email to be used in the DB call so that we ignores case there

Reply:
Was looking at this - update to "const { error, value } = this.validateRequest(req.body);" and then line 29 to "email: value.email," is probably the right approach?